icu4c/source/i18n/unitconverter.h

```diff
@@ -34,15 +34,22 @@ enum U_I18N_API UnitsConvertibilityState {
     UNCONVERTIBLE,
 };
 
+MeasureUnit extractCompoundBaseUnit(const MeasureUnit &source, const ConversionRates &conversionRates,
+                                    UErrorCode &status);
+
 UnitsConvertibilityState U_I18N_API checkConvertibility(const MeasureUnit &source,
                                                         const MeasureUnit &target,
                                                         const ConversionRates &conversionRates,
                                                         UErrorCode &status);
 
 /**
  * Converts from a source `MeasureUnit` to a target `MeasureUnit`.
+ *
+ * NOTE:
+ *    the source and the target must be singular such as `meter` to `mile` or `mile-per-second` to
+ *    `kilometer-per-millisecond`. However, `foot+inch` is not permitted.
```

Comment:
"must not be a MIXED unit", going for the terminology in the enum (and the spec, I hope/expect, if memory serves).

Reply:
(I'm proposing "must not be a MIXED unit" instead of "must be singular".)

Reply:
Done!